src/SonarScanner.MSBuild.Common/RegexConstants.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * SonarScanner for .NET
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto: info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+using System;
+
+namespace SonarScanner.MSBuild.Common
+{
+    public static class RegexConstants
+    {
+        public static TimeSpan DefaultTimeout => TimeSpan.FromSeconds(1);
```

Comment:
1 sec sound way too long for me. It's usually used for properties or file names = reasonably short data that should be a pice of cake for regex to evaluate. I used `TimeSpan.FromMilliseconds(100)` for the paths in sonar-dotnet